R/z_print.R

```diff
@@ -82,6 +82,14 @@ print.animint <- function(x, ...){
 ##'     scale_size_animint(pixel.range=c(2,20), breaks=10^(4:9)))
 animint <- function(...){
   L <- list(...)
+  
+  # Generate a list of duplicate named arguments
+  duplicate_args <- names(L)[duplicated(names(L))]
+  
+  if (length(duplicate_args) > 0) {
+    stop(paste("Error: Duplicate arguments are passed to animint. Duplicate arguments found :", 
```

Comment:
looks like a good start thanks!
please remove Error: (that will show up anyway when you use stop) and please remove space before colon
"found :" -> "found:"
also next time can you please commit and push the test code so that we can see that it fails on github actions, before you push the code which makes the test pass?

Reply:
Updated the error message. Next time, I will push the test code first before pushing the code

Reply:
looks good thanks!